src/app/shared/components/category-card/category-card.component.ts

```diff
@@ -11,9 +11,10 @@ import { SetDirections } from '../../store/filter.actions';
 })
 export class CategoryCardComponent implements OnInit {
   @Input() workshopsCount: number;
-  @Input() isEditMode: boolean;
+  @Input() isEditMode: true;
```

Comment:
true is not a typo of data:) if you want to set a default value  then it should be @Input() isEditMode = true;
if you want to say the data typo is boolean then you should remain the previous version